apps/www/registry/default/ui/alert-dialog.tsx

```diff
@@ -101,10 +101,10 @@ AlertDialogDescription.displayName =
 const AlertDialogAction = React.forwardRef<
   React.ElementRef<typeof AlertDialogPrimitive.Action>,
   React.ComponentPropsWithoutRef<typeof AlertDialogPrimitive.Action>
->(({ className, ...props }, ref) => (
+>(({ asChild, className, ...props }, ref) => (
   <AlertDialogPrimitive.Action
     ref={ref}
-    className={cn(buttonVariants(), className)}
+    className={cn(!asChild && buttonVariants(), className)}
```

Comment:
Don't you need to pass `asChild` down to [AlertDialogPrimitive.Action](https://www.radix-ui.com/primitives/docs/components/alert-dialog#action) as well? I'm not sure it does anything, but I think it would be valid.